Base/QTGUI/Resources/UI/qSlicerSettingsStylesPanel.ui

```diff
@@ -68,7 +68,7 @@ p, li { white-space: pre-wrap; }
          <item>
           <widget class="QPushButton" name="RemoveAdditionalStylePathButton">
            <property name="toolTip">
-            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
+            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;https://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
```

Comment:
Considering that
1. the official RFC reference the `http` link [^1]
2. current test seems to look for `http://` to detect if a content is rich text [^2][^3],
3. the `richtexteditor` built in Qt designer default the simplified text editor [64]
4. new version of Qt designer (checked with Qt5 and Qt6) do not include the dtd in generated ui files (but still matching `<ui version="4.0">` like the Slicer ones), 


I instead suggest we simplify the UI files by re-creating the content using Qt designer. This is required because in case it need to be converted from RichText to HTML.

```
Base/QTGUI/Resources/UI/qSlicerSettingsStylesPanel.ui
Base/QTGUI/Resources/UI/qSlicerSettingsViewsPanel.ui
Modules/Loadable/Units/Resources/UI/qSlicerUnitsSettingsPanel.ui
Modules/Loadable/Units/Widgets/Resources/UI/qMRMLUnitWidget.ui
```

Last, the `QTextDocument::toHtml` function still reference the `http` link. [^5]

[^1]: https://www.w3.org/TR/html401/strict.dtd  (note also that the link redirect to `https`)

[^2]: https://github.com/qt/qttools/blob/ce05219da0750a3be429b396ea0d13980aa14d70/src/designer/src/lib/shared/richtexteditor.cpp#L848-L856

[^3]: https://github.com/qt/qtbase/blob/9db7cc79a26ced4997277b5c206ca15949133240/tests/benchmarks/gui/text/qtextdocument/main.cpp#L48-L52

[^4]: https://github.com/qt/qttools/blame/ce05219da0750a3be429b396ea0d13980aa14d70/src/designer/src/lib/shared/richtexteditor.cpp#L70

[^5]: https://github.com/qt/qtbase/blob/32e24e011ec206f99c61b981cabb6ed5fe06a57b/src/gui/text/qtextdocument.cpp#L2358-L2363

Reply:
These https occurrences must not be changed. They are not for retrieving content but for identification.

Reply:
Changes have been reverted here.